test/default-xpring-client-test.ts

```diff
@@ -32,6 +33,23 @@ const fakeErroringNetworkClient = new FakeNetworkClient(
   FakeNetworkClientResponses.defaultErrorResponses,
 )
 
+/**
+ * Convenience class for creating more specific Error objects to mimick grpc error responses.
+ */
+class FakeServiceError extends Error {
```

Comment:
Maybe call this FakeGRPCError since it's mirroring gRPC semantics?

Reply:
agreed!